CourseProgram.py

```diff
@@ -35,7 +35,7 @@ def main():
          print(f"Seats left: {python.get_seats()}")
          print()
       else:
-         print(f"Sorry {register_Student.get_name()}, registration is closed for {python.get_name()}")
```

Comment:
since we only set `register_Student` in the if branch (not the else branch), it's gonna contain the name of the last student and output this:

```
pmc@pmc-desktop:~/src/Guerra_Exam1$ python3 CourseProgram.py 
Student Name: John
Course Name: MIS 4322 - Advanced Python
CRN: 250309
Seats left: 3

Student Name: James
Course Name: MIS 4322 - Advanced Python
CRN: 250309
Seats left: 2

Student Name: Jill
Course Name: MIS 4322 - Advanced Python
CRN: 250309
Seats left: 1

Student Name: Jack
Course Name: MIS 4322 - Advanced Python
CRN: 250309
Seats left: 0

Sorry Jack, registration is closed for MIS 4322 - Advanced Python
```

We want that line to read "Sorry **Joanne**", not Jack. I changed this line to use the `student` variable that contains the current `for` iteration, since `student` just contains the name anyways.